draw.c

```diff
@@ -151,7 +151,8 @@ void draw_splits(struct state *s, int w, int h, int *y, int off, struct split *s
 		}
 
 		// For the active split, we want to display the delta as soon as it goes over gold
-		if (active && (s->split_time > times.best)) {
+		bool dyn_delta = active && (s->split_time > times.best);
```

Comment:
I see the issue this fixes, but it doesn't feel like a very elegant solution. I wonder if there's a better way to fix this?

Reply:
The other way I considered was a complete refactor, that didn't seem like such a good idea at 1 am. I'll think a bit more about this.

Reply:
I'll take a look later on and see if I can think of anything. If I can't, this is fine

Reply:
I changed it to something better that should be equivalent.

Reply:
The new thing is certainly better, but there's now some duplicate logic (see line 162 vs 191); I think this can probably be merged to remove the outer `cur == UINT64_MAX` check entirely

Reply:
Okay, I've figured out a solution - I'll merge this in and apply it